docs/guide/tools/geometry.md

```diff
@@ -0,0 +1,159 @@
+tools can be manipulated via touch on a touchscreen or via the keyboard. Drawing happens
+through the mouse or stylus.
+
+## Setsquare overview
+
+With the setsquare you can measure distances and angles and draw straight lines as well as
+perpendicular lines to already drawn lines.
+
+![Setsquare-Overview](../../img/guide/setsquare_overview.png)
+
+The longest side of the setsquare is used for measuring elements of the document and drawing
+straight lines aligned to it.
+There are marks (in green) parallel to the longest side of the setsquare, that allow
+measuring distances from that side.
+
+Angular marks (in purple) help measuring angles. There is also a display of the current angle
+(in tenth of degrees) between the longest side of the setsquare and a horizontal line.
+The number always looks up. On the picture this angle is 10.0 degrees.
+
+The shorter two sides of the setsquare are not used for drawing lines aligned to them. Instead
+they are used for drawing lines to the middle point of the longest side of the setsquare.
+
+## Compass overview
+
+The compass serves for transporting distances and angles, for drawing circle arcs and sectors and
+for geometric constructions. The compass looks like a wheel.
+
+![Compass-Overview](../../img/guide/compass_overview.png)
+
+The outline of the compass is used for drawing circles, circle arcs and (when "fill" is enabled)
+circle sectors.
+
+There is a marked radius with measuring marks, along which lines can be drawn. The radius of the
+compass/drawn circles can be seen from the last displayed measuring mark.
+
+Angular marks (in purple) help measuring angles. Like for the setsquare there is a display of the
+current angle.
+
+## Units, distinguished point and axis
+
+For all geometry tools lenths are measured in centimeters and angles are measured in degrees.
+
+Each geometry tool has a distinguished point and a distinguished axis through that point.
+The distinguished point is used as a rotation and scaling center when rotating/scaling via keyboard.
+Angles are measured between horizontal axis and distinguished axis.
+
+- **Setsquare**: The distinguished axis and point are the longest side of the setsquare and its
+  midpoint, respectively.
+- **Compass**: Its distinguished point is the center of the compass wheel. The distinguished axis
+  is a radius that carries measuring marks and can be used to draw a radius or part of it.
+
+Using the keyboard (`m` key) the current distinguished point can be marked with an "x"-like shape
+that is added to the page. For example when drawing a circle the center using the compass,
+the center of the circle can be marked like that.
+
+## Setup and activation
+
+In order to setup the geometry tools, check your Display DPI Calibration in the Zoom panel of menu
+`Edit > Preferences` to ensure that the length on your screen matches the physical length. Set
+the zoom to 100% to profit from the calibration.
```

Comment:
Should set the zoom to 100% before calibration. Might also want to suggest using a physical ruler to compare the length.

Reply:
I don't think it is needed to set the zoom to 100% before calibration. Moreover it is already suggested there to use a physical ruler.

![grafik](https://github.com/xournalpp/xournalpp.github.io/assets/40485433/302e8b60-edcb-41ef-a184-5e75a657dcfc)